conans/test/integration/command/test_profile.py

```diff
@@ -1,7 +1,31 @@
+import os 
+
 from conans.test.utils.tools import TestClient
 
 
 def test_profile_path():
     c = TestClient()
     c.run("profile path default")
     assert "default" in c.out
+
+
+def test_ignore_paths_when_listing_profiles():
+    c = TestClient()
+    ignore_paths = ['.DS_Store']
```

Comment:
It is perfectly fine to start with test as simple as possible, like managing only a single ``.DS_Store``, and not needing a ``for``